backend/models/dynamodb/Entity.js

```diff
@@ -0,0 +1,87 @@
+
+import DynamoDB from "@klaudsol/commons/lib/DynamoDB"
+import { 
+  DYNAMO_DB_TYPES, 
+  DYNAMO_DB_INDEXES, 
+  DYNAMO_DB_TABLE, 
+  DYNAMO_DB_ORDER 
+} from "@/constants";
+
+export default class Entity {
+  // Gets all items where entity = entity_type_slug (eg. tech)
+  // and if type = content
+  // Sample: api/tech
+  // TODO: Add pagination
+  // TODO: Add filtering
+  // TODO: Add search
+  static async whereEntityTypeSlug({ entity_type_slug, order }) {
```

Comment:
We can make order as optional for brevity.

```suggestion
  static async whereEntityTypeSlug({ entity_type_slug, order = null }) {
```